tests/phpunit/tests/rest-api/rest-widgets-controller.php

```diff
@@ -853,6 +853,72 @@ public function test_create_item_second_instance() {
 		);
 	}
 
+	/**
+	 * @ticket 41683
+	 */
+	public function test_create_and_delete() {
+		$this->setup_widget(
+			'text',
+			1,
+			array(
+				'text' => 'Custom text test',
+			)
+		);
+		$this->setup_sidebar(
+			'sidebar-1',
+			array(
+				'name' => 'Test sidebar',
+			),
+			array( 'text-1' )
+		);
+
+		$request = new WP_REST_Request( 'POST', '/wp/v2/widgets' );
```

Comment:
@adamziel would be helpful to contributors reading this test if there were inline comments to explain the "create" and "delete". For example, here a new widget is being added. On line 900, the first text widget is being deleted.

Reply:
Good idea, done done!